src/components/BulkEditPane/BulkEditMarkLayer/BulkEditMarkLayer.test.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { MemoryRouter } from 'react-router-dom';
 import { QueryClientProvider } from 'react-query';
-import { render, waitFor, within } from '@testing-library/react';
+import { render, waitFor, within, screen, getByTestId, logDOM } from '@testing-library/react';
```

Comment:
Could you remove unused methods (getByTestId, logDOM)?

Reply:
Sure, done! @artem-blazhko please check it out again